src/components/QuizData.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import '../QuizData.css';
+
+export default class QuizData extends Component {
+  constructor() {
+    super();
+    this.state = {
+      quizzes: [],
+      questions: [],
```

Comment:
We probably don't need to save questions into our state, since we can access them directly through the quiz itself. A simpler state might just store the single quiz we get back when we hit our API endpoint.

```javascript
this.state = {
    quiz: {}
}
```